packages/form-js-playground/src/Playground.js

```diff
@@ -1,22 +1,14 @@
-import JSONView from './JSONView';
```

Comment:
No change shall be necessary here I believe?

Reply:
I'll revert this file, was just using it for testing the feature.